tmt/steps/prepare/__init__.py

```diff
@@ -247,7 +254,14 @@ def go(self) -> None:
             sync_with_guests(
                 self,
                 'push',
-                PushTask(guests=guest_copies, logger=self._logger),
+                PushTask(
+                    logger=self._logger,
+                    result=None,
+                    guest=None,
```

Comment:
`guest` and `guests` ... Could they be renamed? Highly confusing to have them both in single signature

Reply:
Right, I forgot it's still ugly. I'll improve this, can't use the default `__init__` and default values.

Reply:
Agreed, would be better to make the difference more clear.

Reply:
Hopefully improved in 69824309

Reply:
Yeap, thanks!